core/src/main/java/hudson/cli/handlers/JobOptionHandler.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2004-2009, Sun Microsystems, Inc.
```

Comment:
Ok I missed this was a a rename.
Probably should have the date range updated and you added as a contributor, but I think if this was just a rename then we are ok